spec/controllers/admin/admin_logs_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Admin::LogsController, type: :controller do
+  describe 'Logs' do
+    let(:admin_user) { create(:admin_user) }
+    before do
+      login_by_admin_user admin_user
+    end
+
+    describe 'GET /admin/logs/index' do
+      it 'works!' do
+        get 'index'
+        expect(response.status).to eq(200)
+      end
+    end
+
+    describe 'GET /admin/logs/show' do
+      it 'works!' do
+        get :show, params: { path: Rails.root.join('log/test.log') }
```

Comment:
__[RuboCop]__

Rails/FilePath: Please use `Rails.root.join('path', 'to')` instead.


<!-- Comment by [SideCI](https://sideci.com) -->